src/interfaces.ts

```diff
@@ -123,10 +90,21 @@ export interface Connection {
   contract: string;
   list: () => Promise<TableMetadata[]>;
   create: (
-    query: string,
-    options?: CreateTableOptions
+    /** The schema that defines the columns and constraints of the table,
```

Comment:
chainId is gone, it's being sniffed out of signer.provider

Reply:
Nice, this is much cleaner.